src/kits/tracker/ContainerWindow.cpp

```diff
@@ -310,22 +307,13 @@ AddMimeTypeString(BObjectList<BString>& list, Model* model)
 //	#pragma mark - DraggableContainerIcon
 
 
-DraggableContainerIcon::DraggableContainerIcon(BRect rect, const char* name,
-	uint32 resizeMask)
+DraggableContainerIcon::DraggableContainerIcon(rgb_color menuBarColor)
 	:
-	BView(rect, name, resizeMask, B_WILL_DRAW | B_FRAME_EVENTS),
+	BView("DraggableContainerIcon", B_WILL_DRAW | B_FRAME_EVENTS),
```

Comment:
Does it still need frame events? After all, you removed the FrameMoved() function.